src/js/main.js

```diff
@@ -23,7 +21,23 @@
     })//END OF RESULTS
 
 
-
   })//END OF MODULE
+  .controller('Find', ['$http', '$scope', function($http, $scope){
+    var BASEURL = 'http://wishcastr-staging.herokuapp.com/products/';
```

Comment:
You should move this into [a `.constant` definition](https://docs.angularjs.org/api/ng/type/angular.Module#constant) for reuse between Controllers, etc.